packages/orchestration/src/utils/orchestrationAccount.js

```diff
@@ -17,7 +17,7 @@ export const orchestrationAccountMethods = {
   getAddress: M.call().returns(ChainAddressShape),
   getBalance: M.call(M.any()).returns(Vow$(DenomAmountShape)),
   getBalances: M.call().returns(Vow$(M.arrayOf(DenomAmountShape))),
-  send: M.call(ChainAddressShape, AmountArgShape).returns(VowShape),
+  send: M.call(ChainAddressShape, M.arrayOf(AmountArgShape)).returns(VowShape),
```

Comment:
@dtribble please note this change. It was motivated by the interface of cosmos bank MsgSend, and maybe shouldn't bleed into this interface. I'm not sure the story for doing a multi-send on EVM chains of ERC-20's is as straightforward (iiuc, usually a special contract is required for that).

Reply:
That the Cosmos account supports multi-send doesn't seem like sufficient motivation to change this interface. Single can easily be implemented with multi, but not vice-versa.

Since this is an abstraction, we could also optimize the multi transparently underneath.

Reply:
Thanks for weighing in. Is this to say we should revert and remove the multi-send test cases? 

> we could also optimize the multi transparently underneath

Curious what you have in mind here

Reply:
We had a pattern of `foo` and `fooAll` where the "all" was for the version that took an array. Has that gone away?

I think by far the most common case is one send. Coincidentally, a scenario from yesterday desired sending two in a batch: payment for an operation and payment for fees, which may be a common pattern.

Reply:
Ok. I kept `send` as is and implemented `sendAll` in these fixups as it seemed straightforward to tackle now:
- [dbf7268](https://github.com/Agoric/agoric-sdk/pull/9870/commits/dbf7268a7d7b5749002fbfafecefb0a116c05e30)
- [948e1da](https://github.com/Agoric/agoric-sdk/pull/9870/commits/948e1daa7cd0f6c7ebcd3ed2be5f14fdf94e7e34)
- [5e23652](https://github.com/Agoric/agoric-sdk/pull/9870/commits/5e23652fc5484a7a67719025132531bc2a08c1bb)

PTAL @turadg

Reply:
Thanks @turadg.

@dtribble P.S. the other type of "batch/multi send" I anticipate is something that has many recipients - i.o.w. an array of [ChainAddress , AmountArg] pairs.